assets/app/views/_templateopt.html

```diff
@@ -1,4 +1,4 @@
-<div class="template-options" ng-show="hasParameters">
+<div class="template-options" ng-show="hasParameters" ng-form="paramForm">
```

Comment:
We would have a problem with duplicate IDs, but that's not specific to this pull.